querydsl-sql/src/test/java/com/querydsl/sql/SelectBase.java

```diff
@@ -232,6 +232,13 @@ public void Boolean_Any() {
     }
 
     @Test
+    public void Case() {
```

Comment:
@timowest I tried to add an assertion here but H2, Derby, and Hsql seem to order the result differently. Can you tell what I'm missing here? I'll look at it tomorrow if you haven't fixed it.